app.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "MusicStore",
+    "description": "ASP.NET 5 sample app",
+    "logo": "https://raw.githubusercontent.com/aspnet/MusicStore/master/src/MusicStore/wwwroot/Images/home-showcase.png",
```

Comment:
@Praburaj I agree that it's not great. I can take it out if you want. Heroku requires an absolute URL to be able to fetch the image. 

Let me know if you want me to yank it.